### DISCUSSION TITLE
I would like to create a mbtiles service that has global boundaries and places from z0-3 combined with US only data from z0-14.

### DISCUSSION THREAD

**Discussion by schierkolk** [systemed/tilemaker#480] [Q&A]
`cd usa
wget https://download.geofabrik.de/north-america/us-latest.osm.pbf
nohup tilemaker --input us-latest.osm.pbf --output us-latest.mbtiles --config config-openmaptiles.json --process process-openmaptiles.lua &

cd ../world
wget https://planet.osm.org/pbf/planet-latest.osm.pbf
wget https://osmdata.openstreetmap.de/download/water-polygons-split-4326.zip
unzip water-polygons-split-4326.zip
osmium tags-filter planet-latest.osm.pbf w/boundary w/places -o planet-boundary-places.osm.pbf
tilemaker --input planet-boundary-places.osm.pbf --output ../usa/us-latest.mbtiles --merge --config config-openmaptiles-world.json --process process-openmaptiles.lua
`

This is one thing I have tried. Specifying a maxxoom as 3 in the config file. In this iteration the global boundary covers the US data.

I have tried the other way and I get boundary everywhere except for the US.

**Comment by systemed** [comment]
I'd probably do something like this:

- Download Natural Earth boundaries
- Create a custom .json and .lua to build planet-wide tiles at z0-3 using these boundaries and the coastline
- Run tilemaker over this to create an.mbtiles
- Run tilemaker as normal for the US, but with maxzoom set to 4 and with --merge
- Make sure that the mbtiles metadata is set to what you need (by opening the .mbtiles with an SQLite editor)
